src/current/_includes/v23.2/sql/grant-revoke-schema-changes.md

```diff
@@ -1,9 +1,19 @@
 
-This means that [long-running transactions]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#hanging-or-stuck-queries) elsewhere in the system can cause `GRANT` and `REVOKE` operations inside transactions to take several minutes to complete. This can have a cascading effect. When a `GRANT` or `REVOKE` operation inside a transaction takes a long time to complete, it can in turn block all user-initiated transactions being run by your application, since the `GRANT` operation in the transaction has to commit before any other transactions that access role memberships (i.e., most transactions) can make progress.
+This means that [long-running transactions]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#hanging-or-stuck-queries) elsewhere in the system can cause user/role management operations inside transactions to take several minutes to complete. This can have a cascading effect. When a user/role management operation inside a transaction takes a long time to complete, it can in turn block all user-initiated transactions being run by your application, since the user/role management operation in the transaction has to commit before any other transactions that access role memberships (i.e., most transactions) can make progress.
 
-If you would prefer that grant and revoke operations finish rapidly, and do not care whether concurrent transactions will immediately see the side-effects of those operations, set the session variable `allow_role_memberships_to_change_during_transaction` to `true`.
+If you want user/role management operations to finish more quickly, and do not care whether concurrent transactions will immediately see the side effects of those operations, set the [session variable]({% link {{ page.version.version }}/set-vars.md %}) `allow_role_memberships_to_change_during_transaction` to `true`.
 
-When enabled, this session variable means that the grant or revoke operations issued in the current session will only need to wait for the completion of statements in other sessions which do not have that session variable set. To accelerate grant and revoke operations across your entire application, you can set the session variable in all sessions by [passing it in the client connection string]({% link {{ page.version.version }}/connection-parameters.md %}#supported-options-parameters).
+When this session variable is enabled, any user/role management operations issued in the current session will only need to wait for the completion of statements in other sessions where `allow_role_memberships_to_change_during_transaction` is not enabled.
+
```

Comment:
Extra line break here.

Reply:
Ya that was intentional for skim-reading, so the final thing where we say "to speed this up, do X" stands out and is not at the tail end of a longish paragraph